### DISCUSSION TITLE
vuetify.js as it exceeds the max of 500KB.

### DISCUSSION THREAD

**Discussion by ricardovanlaarhoven** [vuetifyjs/vuetify#14804] [Q&A]
### Environment
**Vuetify Version:** 2.6.4
**Vue Version:** 2.6.13
**Browsers:** Chrome 99.0.4844.51
**OS:** Windows 10

### Steps to reproduce
just `npm run build`

### Expected Behavior
No warning

### Actual Behavior
[BABEL] Note: The code generator has deoptimised the styling of ......../node_modules/vuetify/dist/vuetify.js as it exceeds the max of 500KB.

### Reproduction Link
there is no useful reproduction link possible in my opinion

**Comment by KaelWD** [comment]
https://babeljs.io/docs/en/options#compact

There won't be 500KB in a single file if you use https://vuetifyjs.com/en/features/treeshaking/

**Reply by ricardovanlaarhoven** [reply]
This project i'm talking about is created a few months ago using the `vue-cli` and the `vue-cli-plugin-vuetify` 
Should i do something manually for this?

**Reply by KaelWD** [reply]
- Install `vuetify-loader sass@1.32` (doesn't need any configuration)
- Change the vuetify import to `import Vuetify from 'vuetify/lib/framework'`
- Remove any `vuetify.css`/`vuetify.min.css` import
---
Or add `"compact": false` in your babel config if you just want to hide the message.
